### ISSUE TITLE
Generic object responses should use 'any` and not generate a type

### ISSUE THREAD

**Issue by melloware** [anymaniax/orval#308]
## What are the steps to reproduce this issue?

**Orval:** ^6.6.1
**Type:** react-query

Spring Boot has a Health Check endpoint defined as below with just scheme `type = Object`. because its an ANY response as a health check can have different details depending on what you are monitoring like Disk Space ,or Database Health Check etc.

**OpenApiV3.json**
```javascript
"/actuator/health": {
      "get": {
        "tags": [
          "Actuator"
        ],
        "summary": "Actuator web endpoint 'health'",
        "operationId": "health",
        "responses": {
          "200": {
            "description": "OK",
            "content": {
              "*/*": {
                "schema": {
                  "type": "object"
                }
              }
            }
          }
        }
      }
    }
```

A typical response from the Health Check looks like this...

```javascript
{
  "status": "UP",
  "details": {
    "diskSpace": {
      "status": "UP",
      "details": {
        "total": 250685575168,
        "free": 188298530816,
        "threshold": 10485760
      }
    }
  }
}
```
However Orval is generating an empty TypeScript object like this:

```TypeScript
export type Health200 = {};
```

So now in my code I have to cast `Health200` to `any` so I can get to the `status field.

```typescript
if (healthCheck.isSuccess) {
    const result = healthCheck.data as any; // cast as any to get to status
    if (result.status !== "UP") {
        throw new Error('REST API is currently down.');
    }
}
```

**Comment by melloware** [comment]
Here is my OpenAPI JSON to demo the issue:
[melloware-openapi.zip](https://github.com/anymaniax/orval/files/8003446/melloware-openapi.zip)

**Comment by melloware** [comment]
I thought I had fixed it but I just tried 6.6.1 and its still generating...

```ts
export type Health200 = {};
```

So I am reopening this ticket.

**Comment by melloware** [comment]
@CPatchane @anymaniax I may need some direction on where in the code to fix this?  I thought I had fixed it in the right place but clearly not.  Any guidance would be appreciated!

**Comment by anymaniax** [comment]
@melloware I will check when I have a bit of time

**Comment by melloware** [comment]
Thank you and thanks for all your effort this tool is fantastic and getting better every day!

**Comment by anymaniax** [comment]
Also thanks to you and @CPatchane!

**Comment by anymaniax** [comment]
I found the solution is it ok for you if I put this instead `Record<string, any>`

**Comment by melloware** [comment]
Let me try it locally with that.

**Comment by melloware** [comment]
So you are saying it would look like this?

```js
export type HealthPath200 = Record<string, any>;
```

**Comment by anymaniax** [comment]
Exactly

**Comment by melloware** [comment]
OK I am trying it but I am struggling to see how I use the Record.


Right now my code does this...

```ts
if (healthCheck.isSuccess) {
            const result = healthCheck.data as any;
            const newStatus = result.status;
            if (newStatus !== "UP") {
                handleError();
            }
}
```

By casting data to `any` i can get at the `.status` property or any other nested property.  With `Record<string, any>` i am getting compile failures. I don't see how to use the Record?

**Comment by anymaniax** [comment]
normally that should work properly. Weird. Which version of typescript did you have?

The following example is in the typescript [playground](https://www.typescriptlang.org/play?#code/FAYw9gdgzgLgBACwKYEMA2MEGFkgNYBccASkuAE4AmAPLOQJYQDmANHChAJ4B8ccAvHADeAX2ChIsRB0pokAUXLkw5AXAAUASgG9R4+gDMNydJhxk8AOnpQAygFcQIJFCjahwPl+9fw0eOQu9hhqJhjYuFaUKDAo7FDsXADcnj5pflIQSADutrEw9gmCgVDBMJawMYUpaWmGGlm5+YVwAIT8ggBEAKoACp3uqbW1CDJyisrkWjXDXmIiQA)
<img width="387" alt="Screen Shot 2022-02-17 at 19 18 34" src="https://user-images.githubusercontent.com/10516382/154545454-ff16d781-a675-4b1e-a04b-8526acaa45af.png">

**Comment by melloware** [comment]
Oh yeah this works...

```
const result = healthCheck.data as any;
```

My whole goal of this was so i wouldn't have to case `{}` to `any` that the Health200 would just be an `any`.

**Comment by melloware** [comment]
So the code should just read like this...

```ts
if (healthCheck.isSuccess) {
            const newStatus = healthCheck.data.status;
            if (newStatus !== "UP") {
                handleError();
            }
}
```

**Comment by melloware** [comment]
Ahh yep it does work!  I like it!

**Comment by melloware** [comment]
so to repeat this does work fine and I can remove the cast to `any`.

```ts
export type Health200 = Record<string, any>;
```

**Comment by anymaniax** [comment]
Ok I will put this which is more retro compatible and work the same way
```ts 
  export type HealthPath200 = { [key: string]: any };
```

**Comment by anymaniax** [comment]
fixed with `6.6.2`

**Comment by melloware** [comment]
you rock!
